.github/workflows/build.yml

```
@@ -0,0 +1,88 @@
+name: Build app package
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_pkg:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Checkout Dependecies
+      run: |
+        git clone https://github.com/illusion0001/apollo-lib.git -b no-dbglgr --depth 1
+        git clone https://github.com/bucanero/zip.git                         --depth 1
+        git clone https://github.com/bucanero/oosdk_libraries.git             --depth 1
+        git clone https://github.com/bucanero/mxml                            --depth 1
+        git clone https://github.com/Al-Azif/ps4-skeleton.git                 --depth 1
+
+    - name: Set env vars
+      run: |
+        echo commit_ver=1.0`git rev-list HEAD --count`         >> $GITHUB_ENV
+        echo pkg_file=IV0000-APOL00004_00-APOLLO0000000PS4.pkg >> $GITHUB_ENV
+
+    - name: Download OpenOrbis Toolchain
+      run: |
+        curl -fLO https://github.com/illusion0001/OpenOrbis-PS4-Toolchain/releases/download/0.0.1.416/toolchain.tar.gz
+        tar xf toolchain.tar.gz
+        echo "OO_PS4_TOOLCHAIN=$GITHUB_WORKSPACE/OpenOrbis/PS4Toolchain" >> $GITHUB_ENV
+        cp build_rules.mk OpenOrbis/PS4Toolchain/build_rules.mk
+
+      # >10 is borked, for now
+    - name: Install LLVM and Clang 
+      uses: KyleMayes/install-llvm-action@v1
+      with:
+        version: "10.0"
```

Why not do it this way?
```suggestion
      # >10 is borked, for now
    - name: Install LLVM and Clang 
      run: |
      	sudo apt-get -y update
      	sudo apt-get -y install llvm clang
```